lib/helpers/ContentRenderUiHelper.ts

```diff
@@ -14,7 +14,11 @@ export class ContentRenderUiHelper extends UiBaseLocators {
     await this.page.goto(umbracoConfig.environment.baseUrl + contentURL);
   }
 
-  async doesContentRenderValueHaveText(text: string) {
-    return await expect(this.contentRenderValue).toContainText(text);
+  async doesContentRenderValueHaveText(text: string, isEqual: boolean = false) {
```

Comment:
```suggestion
  async doesContentRenderValueContainText(text: string, isEqual: boolean = false) {
```
Since isEqual is set to false, the naming should be ContainText